docs/tutorial.rst

```diff
@@ -1588,11 +1581,13 @@ and that rates of migration since the split between the populations are both zer
         NA=5000, N1=4000, N2=1000, T=1000, M12=0, M21=0
     )
 
-We'll simulate 10 chromosomes from each of the populations using the ``msprime`` engine.
+We'll simulate 5 diploids from each of the populations using the
+``msprime`` engine (the populations in this generic model are named
+``pop1`` and ``pop2``).
 
 .. code-block:: python
 
-    samples = model.get_samples(10, 10)
```

Comment:
Is this right? Wouldn't the old code be sampling pop_0 and pop_1

Reply:
Nope, looks like the names in the `IsolationWithMigration` model are `pop1`, `pop2`, and `ancestral`.